### PR TITLE
Single_deploy

### PR DESCRIPTION
**Pull request by vtlrazin**
Single YAML deployment file with CRD, namespace, rbac, and mpi-operator sections.
This will simplify deployment for ANSIBLE-based installations as well as for other installations.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mpi-operator/136)

### CODE REVIEW

**Comment by vtlrazin** [comment]
I signed it!

File: deploy/mpi-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+#      MPI-operator container deployment set
+# --------------------------------------------------
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: mpijobs.kubeflow.org
+spec:
+  group: kubeflow.org
+  version: v1alpha2
```
Comment (terrytangyuan) [inline deploy/mpi-operator.yaml]: This is only the v1alpha2 CRD. Perhaps we should only combine the yaml files besides the CRDs so we don't need to keep two files with a lot of overlapped content, e.g RBAC, etc.

Comment (vtlrazin) [reply deploy/mpi-operator.yaml]: CRD with v1alpha1 is supported by mpioperator/mpi-operator:latest or customer must use mpioperator/mpi-operator:v1alpha1 version?

Comment (terrytangyuan) [reply deploy/mpi-operator.yaml]: Sorry I somehow missed this question. No, v1alpha1 would not work with the latest images.

Comment (vtlrazin) [reply deploy/mpi-operator.yaml]: So, we can leave only the v2 version on the master branch. But single yaml deployment file will make deployment easier.

Comment (vtlrazin) [reply deploy/mpi-operator.yaml]: Do you know how to resolve LGTM label issue?

Comment (terrytangyuan) [reply deploy/mpi-operator.yaml]: cc @rongou to double check whether we need to make v1 version of it and whether broken existing links are fine.

Comment (rongou) [reply deploy/mpi-operator.yaml]: Yeah I think it's fine to ignore v1.

Comment (vtlrazin) [reply deploy/mpi-operator.yaml]: Hi,
How will we continue? Do I need to do something?

Comment (terrytangyuan) [reply deploy/mpi-operator.yaml]: Thank you! Approving now.

**Pull request merged**